src/chapters/objects-and-math/background.rst

```diff
@@ -0,0 +1,132 @@
+Objects and Why They Matter
+===========================
+
+.. index:: ! object, ! key/value pair, ! key
+
+So far we have learned a lot about arrays, which are data structures that can hold many values.
+**Objects** are also data structures that can hold many values. 
+
+Unlike arrays, each value in an object has a name or **key** for reference purposes.
+The pairing between a key and its value is called a **key/value pair**. 
+
+To recap, objects store as many key/value pairs as needed and each value needs a key. Without a key, the value cannot be accessed or modified by the programmer.
+
+.. figure:: figures/object.png
```

Comment:
This figure is the size of the other figures in the book. What are your thoughts on sizing down?

Reply:
Best practice is to size figures so they and their relevant text can be viewed without scrolling.

Reply:
Let's remember to bring this up on Tuesday! The figure and the text can be viewed on the same screen, but maybe the team wants to set a standard size for figures to avoid figures being too big.

Reply:
Ok.